vars/Debian.yml

```diff
@@ -20,12 +20,10 @@ __php_conf_paths:
   - /etc/php5/cli
 
 __php_extension_conf_paths:
-  - /etc/php5/fpm/conf.d
-  - /etc/php5/apache2/conf.d
-  - /etc/php5/cli/conf.d
+  - /etc/php5/mods-available
```

Comment:
Whilst this PR implements `php5enmod` which is Debian/Ubuntu specific, you don't need to manage all possible Debian/Ubuntu specific `php_extension_conf_paths`. 

All installed PHP sapi's will be automatically linked, with the correct priority when the module is enabled via `php5enmod`.